lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -317,6 +317,22 @@ sub register {
             my ($c, $group) = @_;
             return $c->tag('li', $c->link_to($group->name => $c->url_for('group_overview', groupid => $group->id)));
         });
+
+    $app->helper(
+        comment_icon => sub {
+            my ($c, $jobid, $comment_count) = @_;
+            return '' unless $comment_count;
+
+            return $c->link_to(
+                $c->url_for('test', 'testid' => $jobid) . '#comments' => sub {
+                    $c->tag(
+                        'i',
+                        class => 'test-label label_comment fa fa-comment',
+                        title => $comment_count . ($comment_count != 1 ? ' comments available' : ' comment available')
+                      ),
+                      ;
+                });
+        });
```

Comment:
Oh, you mean moving out the whole set of icons (including bugs and label)? I can also do that but I guess this doesn't speak against merging this part already.

Note that the same approach can not be used for "Last 500 finished jobs" because it uses AJAX and JavaScript - and this is actually not a bad thing because the table is a candidate to be made server-side (especially when displaying more meta-data is required).